transactions/add-account/2024/aug-27/create_account.cdc

```diff
@@ -0,0 +1,38 @@
+import FlowToken from 0x1654653399040a61
+import FungibleToken from 0xf233dcee88fe0abe
+
+transaction(publicKeys: [String], value: UFix64) {
+
+	prepare(signer: AuthAccount) {
+		let acct = AuthAccount(payer: signer)
+
+		for i, publicKey in publicKeys {
+			let key = PublicKey(
+				publicKey: publicKey.decodeHex(),
+				signatureAlgorithm: SignatureAlgorithm.ECDSA_P256,
+			)
+
+			acct.keys.add(
+				publicKey: key,
+				hashAlgorithm: HashAlgorithm.SHA2_256,
+				weight: 1000.0,
+			)
+		}
+
+		// get the receiver from the new account
+        self.tokenReceiver = acct
```

Comment:
This variable is not defined as far ad i can see

Reply:
:( added the definition of the variable